src/Web/Avalonia.Web.Blazor/webapp/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const prod = process.env.NODE_ENV == 'production';
+
+module.exports = {
+    mode: prod ? "production" : "development",
+    devtool: 'source-map',
+    target: ["web", "es2020"],
```

Comment:
Are we sure that "es2020" is supported by all browsers where webassembly is supported?

Reply:
No, it's not. I will try to lower it down.
Seems like I raised it after some experiments.